src/scintillaeditor.cpp

```diff
@@ -152,6 +156,14 @@ ScintillaEditor::ScintillaEditor(QWidget *parent) : EditorInterface(parent)
 	c = qsci->standardCommands()->boundTo(Qt::Key_Insert | Qt::CTRL);
 	c->setAlternateKey(0);
 
+	QShortcut *shortcutCalltip;
+	shortcutCalltip = new QShortcut(Qt::CTRL | Qt::SHIFT | Qt::Key_Space, this);
+	connect(shortcutCalltip, &QShortcut::activated, [=]() { qsci->callTip(); });
+
+	QShortcut *shortcutAutocomplete;
+	shortcutAutocomplete = new QShortcut(Qt::CTRL | Qt::Key_Space, this);
```

Comment:
These two should probably be `Qt::META` on macOS, to correctly bind to Ctrl, as Qt:CTRL binds to the Cmd key, which is not what we want to use (since Cmd-space is Spotlight/Launchbar etc. search)

Reply:
Updated.